src/utils/dappMailer.js

```diff
@@ -243,9 +243,9 @@ module.exports = {
       title: 'Milestone review requested',
       image: 'Giveth-review-banner-email.png',
       text: `
-        <p><span style="line-height: 33px; font-size: 22px;">Hi ${data.user}</span></p>
+        <p><span style="line-height: 33px; font-size: 22px;">Hi ${data.user(0).toUpperCase()+data.user.slice(1)}</span></p>
         <p>
-          The milestone <em>${data.milestoneTitle}</em> to the campaign <em>${data.campaignTitle}</em> has been marked as completed by the milestone owner.
+          The milestone <em>${data.milestoneTitle(0).toUpperCase()+data.milestoneTitle.slice(1)}</em> to the campaign <em>${data.campaignTitle(0).toUpperCase()+data.campaignTitle.slice(1)}</em> has been marked as completed by the milestone owner.
```

Comment:
Here too, you can use data.milestoneTitle[0].toUpperCase() .... 
Also the code has eslint problem, you should add spaces before plus sign
Install some eslint plugin on you IDE